Multidb support - Take 2 (#2)
adamalton/djangae-migrations: README.md

	...
```

This will raise `massmigration.exceptions.RequiredMigrationNotApplied` if the function is called before the migration is applied.

### Multiple DBs Support

> When running on multiple databases, by default, the utilities assume that the migration is ran on all the databases specified by the Migration `allowed_db_aliases` attribute.

Reviewer: ```suggestion
When running on multiple databases, by default, the utilities assume that the migration must have been applied on all the databases specified by the Migration's `allowed_db_aliases` attribute.
```
